recipe/meta.yaml

```diff
@@ -11,40 +11,41 @@ source:
     - test_opendap2_url.patch
 
 build:
-  number: 4
+  number: 5
   skip: True  # [win or py3k]
   detect_binary_files_with_prefix: true
 
 requirements:
   build:
+    - {{ compiler('fortran') }}
+    - {{ compiler('c') }}
+  host:
     - python
-    - gcc
-    - numpy 1.11.*
+    - numpy 1.14.*
```

Comment:
I did not check the hdf5 incompatibility when I added this change.
I assumed it was an issue with Fortran extensions that we are having in other builds and it may as well happen here too. However, to fix the `hdf5` pin issue I just sent https://github.com/conda-forge/hdfeos5-feedstock/pull/19